+bids/layout.m

```diff
@@ -391,7 +396,7 @@ function tolerant_message(tolerant, msg)
                             % obtained from an external scan and/or study
                             subject.perf(j).m0type = 'single_value';
                             subject.perf(j).m0explanation = 'this is a single estimated M0 value, e.g. when the M0 is obtained from an external scan and/or study';
-                            subject.perf(j).m0_value = subject.perf(j).meta.M0;                            
+                            subject.perf(j).m0_value = subject.perf(j).meta.M0Estimate;                            
```

Comment:
@HenkMutsaerts
there is no bids-example that covers the usecas "M0Type":"Separate", is there?
this would help with the tests: https://github.com/bids-standard/bids-matlab/blob/dev/tests/test_bids_query_asl.m

Reply:
Good point, let me check. If we don't have this example I can easily create one for you

Reply:
See [example asl002](https://github.com/bids-standard/bids-examples/tree/master/asl002/sub-Sub103/perf)

Reply:
ah sorry, I meant "Estimate" not "Separate".